lib/vagrant/action/builtin/mixin_synced_folders.rb

```diff
@@ -98,11 +98,6 @@ def save_synced_folders(machine, folders, opts={})
           end
 
           folder_data = JSON.dump(folders)
-
-          # Scrub any register credentials from the synced folders
-          # configuration data to prevent accidental leakage
-          folder_data = Util::CredentialScrubber.desensitize(folder_data)
```

Comment:
I think this PR should also maybe remove this line that desensitizes the synced_folder export. Passwords are (and should) never stored be as part of folder data. I think there is also a case to be made about desensitizes information here can lead to leaking of credentials. For example if an exported folder is named "vagrant" and the users password is "vagrant", the synced_folder cache will show "****" in place of the folder name, indicating that it is also password.

But, maybe I'm missing some other consideration here?